docker-compose/postgres/db_init.sql

```diff
@@ -19275,3 +19275,18 @@ VALUES (
 -- PostgreSQL database dump complete
 --
 
+ALTER TABLE dev_cac.dimensions
```

Comment:
@pratikmishra356 when we do the flip, we will have to regenerate the data in this file right?

Reply:
yes , we will regenerate the data along with my second pr